charts/kubernetes/orchestrator/charts/serverless-knative/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+name: serverless-knative
+description: |
+  A Helm chart to install the serverless infrastructure
+  on OpenShift or Kubernetes. Use the value openshift.enable or kubernetes.enable
+type: application
+version: 0.1.0
+appVersion: "1.0.0"
+
+dependencies:
+  - name: openshift
```

Comment:
I first commented on a file below, but seeing this now - would be simpler to remove the openshift if there are no shared deps needed for the k8s installation.